draft-nygren-httpbis-httpssvc.md

```diff
@@ -0,0 +1,791 @@
+{{!RFC6066}}, both of which confer substantial performance and privacy
+benefits when used in combination with HTTPSSVC records.
+
+To realize the greatest privacy benefits, this proposal is intended for
+use with a privacy-preserving DNS transport (like DNS over TLS
+{{!RFC7858}} or DNS over HTTPS {{!DOH=I-D.ietf-doh-dns-over-https}}),
+and with the "SNI" Alt-Svc Parameter
+{{!AltSvcSNI=I-D.bishop-httpbis-sni-altsvc}}.  However, performance
+improvements, and some modest privacy improvements, are possible without
+the use of those standards.
+
+This RRType could be extended to support schemes other than "https".
+Any such scheme MUST have an entry under the HTTPSSVC RRType in the IANA
+DNS Underscore Global Scoped Entry Registry {{!Attrleaf=I-D.ietf-dnsop-attrleaf}}
+The scheme SHOULD have an entry in the IANA URI Schemes Registry {{!RFC7595}}.
+The scheme SHOULD be one for which Alt-Svc is defined.
+
+
+
+# Security Considerations
+
+Alt-Svc Field Values are intended for distribution over untrusted
+channels, and clients are REQUIRED to verify that the alternative
+service is authoritative for the origin (Section 2.1 of {{!AltSvc}}).
+Therefore, DNSSEC signing and validation are OPTIONAL for publishing
+and using HTTPSSVC records.
+
+TBD: expand this section in more detail.  In particular:
+
+* Any additional tracking risks introduced by the sni= parameter.
+  (Perhaps incorporating text from {{!AltSvcSNI=I-D.bishop-httpbis-sni-altsvc}}).
+* ...
+
+
+# IANA Considerations
+
+Per {{?RFC6895}}, please add the following entry to the data type
+range of the Resource Record (RR) TYPEs registry:
+
+| TYPE     | Meaning                | Reference       |
+| ------   | ---------------------- | --------------- |
+| HTTPSSVC | HTTPS Service Location | (This document) |
+
+Per {{?Attrleaf}}, please add the following entries to the DNS Underscore
+Global Scoped Entry Registry:
+
+| RR TYPE   | _NODE NAME | Meaning           | Reference       |
+| --------- | ---------- | ----------------- | --------------- |
+| HTTPSSVC  | _https     | Alt-Svc for HTTPS | (This document) |
+| HTTPSSVC  | _http      | Alt-Svc for HTTP  | (This document) |
+
+
+Per {{?AltSvc}}, please add the following entries to the HTTP Alt-Svc
+Parameter Registry:
+
+| Alt-Svc Parameter | Meaning              | Reference       |
+| ----------------- | -------------------- | --------------- |
+| pri               | Relative priority    | (This document) |
+|                   | of Alt-Svc records   |                 |
+| sni               | SNI value to send    | (This document) |
+| esnikey           | Encrypted SNI key    | (This document) |
+
+
+# Acknowledgements and Related Proposals
+
+There have been a wide range of proposed solutions over the years to
+the "CNAME at the Zone Apex" challenge proposed.  These include
+{{?I-D.draft-bellis-dnsop-http-record-00}},
+{{?I-D.draft-ietf-dnsop-aname-03}}, and others.
+
+
+# Appendix
```

Comment:
This is titled "Appendix," but is a top-level section in the body.  I'd suggest making each of these subsections an actual and separate appendix.

Reply:
Fixed in #12

Reply:
Addressed in #12